tests/e2e/TestAppBundlesApi.cs

```diff
@@ -71,5 +71,30 @@ public async void AppBundles_GetAll()
             }
         }
 
+        [Fact]
+        [Order(Weight = 1.3)]
+        public async void AppBundles_GetAllAlias()
+        {
+            using (Fixture.StartTestScope())
+            {
+                var list = await Fixture.DesignAutomationClient.GetAllItems(async page => await Fixture.DesignAutomationClient.GetAppBundleAliasesAsync(this.app.Id, page));
+                Assert.Contains(list, e => e.Id == "latest");
+                Assert.Contains(list, e => e.Id == "$LATEST");
+                Assert.Distinct(list);
+            }
+        }
+
+        [Fact]
+        [Order(Weight = 1.4)]
+        public async void AppBundles_GetAllVersion()
+        {
+            using (Fixture.StartTestScope())
+            {
+                var list = await Fixture.DesignAutomationClient.GetAllItems(async page => await Fixture.DesignAutomationClient.GetAppBundleVersionsAsync(this.app.Id, page));
+                Assert.Contains(1, list);
+                Assert.Contains(2, list);
+                Assert.Distinct(list);
+            }
```

Comment:
I guess we are adding the missed tests before?

Reply:
Yes, some more tests..